test/helpers/data/aggregated-data-helper.js

```diff
@@ -230,6 +230,15 @@ module.exports.selectIdsForWorkloadOwner = function () {
   return promise
 }
 
+module.exports.getTeamId = function () {
```

Comment:
To me getTeamId implies you are getting the Id of a specific team.  Perhaps rename this.  Maybe to getAnyExistingTeamId? Or something else sensible.